### PR TITLE
NXP-25872: Stabilize Functional Tests

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: package.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
     "postbuild": "npm-run-all fix:*",
     "fix:elements": "replace '/elements/' '${window.location.pathname}/' target/classes/web/nuxeo.war/ui/*.js",
     "fix:ui_elements": "replace '/node_modules/@nuxeo/nuxeo-ui-elements/' '${window.location.pathname}/' target/classes/web/nuxeo.war/ui/*.js",
+    "fix:viewers": "replace '\\${window.location.protocol}//?\\${window.location.host}\\${window.location.pathname}/(viewers|widgets)/[\\w\\d-]+.js' '${window.location.protocol}/${window.location.host}${window.location.pathname}/vendor/' target/classes/web/nuxeo.war/ui/vendors~elements.bundle.js",
```
Comment (nelsonsilva) [inline package.json]: protocol, host and path shouldn't be affected so we should be able to remove them from the regex

Comment (nelsonsilva) [reply package.json]: also, I'd rather go with *.js since number of bundles and their names may change

Comment (Gabez0r) [reply package.json]: > protocol, host and path shouldn't be affected so we should be able to remove them from the regex

Keep in mind that I'm still keeping them on the new string, I'm just using it to make the regex more specific and try to only cover the cases where with use importMeta. I think just blindly replacing `\/(viewers|widgets)\/[\w\d-]+\.js` with `/vendor/` as a great potential to break stuff in the future...

**Comment by nelsonsilva** [comment]
👍

**Pull request merged**